Mathlib/Data/Set/Quot.lean

```diff
@@ -0,0 +1,77 @@
+/-
+Copyright (c) 2025 Pim Otte. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Pim Otte
+-/
+
+import Mathlib.Data.Set.Card
+
+/-!
+# Quotients as Sets
+
+## Main definitions
+
+* `Quot.toSet` interprets an equivalence class as a set.
+* `Set.Represents` says that a set represents a set of quotients if it contains exactly one element
+  from each quotient.
+-/
+
+variable {α : Type*} {r : α → α → Prop}
+
+/-- Convert an equivalence class to a `Set`. -/
+def Quot.toSet (c : Quot r) : Set α := {v | Quot.mk r v = c}
+
+instance : SetLike (Quot r) α where
```

Comment:
Can you add the other theorems that the module docstring for `SetLike` prescribes? For instance, that `v ∈ c \iff Quot.mk r v = c`?

Reply:
I added the mem and ext lemma's. If the copy lemma's also apply here I'm not really sure how to implement that, so if it does, I'd appreciate a pointer:)